src/main/java/controller/RacingCarController.java

```diff
@@ -1,20 +1,27 @@
 package controller;
 
+import domain.RacingCar;
 import domain.RacingCarGame;
 import view.InputView;
 import view.OutputView;
 
+import java.util.List;
+
 public class RacingCarController {
 
     public static void main(String[] args) {
-        int createCarCount = InputView.inputCarCount();
+        String[] createCarNames = InputView.inputCarNames();
         int tryCount = InputView.inputTryCount();
 
-        RacingCarGame racingCarGame = new RacingCarGame(createCarCount);
+        RacingCarGame racingCarGame = new RacingCarGame(createCarNames);
 
         System.out.println("실행 결과");
         for (int i = 0; i < tryCount; i++) {
-            OutputView.printCarsPositionStatus(racingCarGame.playAndReturnCars());
+            List<RacingCar> racingCars = racingCarGame.playAndReturnCars();
+            OutputView.printCarsNameAndPositionStatus(racingCars);
         }
+
+        List<RacingCar> cars = racingCarGame.getCars();
+        OutputView.printWinners(RacingCarGame.getWinners(cars));
```

Comment:
static 메소드 사용을 안할 수는 없을까요?
racingCarGame 객체에게 작업을 위임해 놓고, 결과는 해당 객체가 아닌 static 메소드를 통해 받아오는 느낌이 들어서요. :) 
```suggestion
OutputView.printWinners(racingCarGame.getWinners());
```